cal/cal/static/css/calendaranalytics.scss

```diff
@@ -20,6 +20,10 @@ button {
   margin: 2px;
 }
 
+a[ng-click]{
```

Comment:
What is this for? Are you doing this because hovering over a link with ng-click doesn't show the hand? If so, may I suggest you try `href=""`? Generally, using attribute names in css selectors is messy when adding a class will do.